perceval/backends/cliffords2017.py

```diff
@@ -43,9 +44,11 @@ class CliffordClifford2017Backend(Backend):
     def prob_be(self, input_state, output_state, n=None, output_idx=None):
         raise NotImplementedError(f'Cannot use prob_be on {self.name}')
 
-    def sample(self, input_state):
+    def sample(self, input_state: Union[BasicState, StateVector]) -> BasicState:
         if isinstance(input_state, StateVector):
-            input_state = input_state.sample()
+            if len(input_state) != 1:
+                raise RuntimeError(f"{self.name} cannot sample with a mixed state input")
```

Comment:
It's not mixed state input (= SVDistribution) that lead to a problem, but superposed pure states input (= StateVector of len > 1). The assertion is right, not the message

Reply:
Right, thanks. Fixed!